tests/commands/merge_user.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import pytest
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_merge_user_nofrom(capfd, nobody, member2):
+    with pytest.raises(CommandError) as e:
+        call_command('merge_user', 'not_a_user', 'member2')
+    assert "User not_a_user does not exist" in str(e)
```

Comment:
Missing tests for not passing enough arguments or no arguments at all to `merge_user`?